lib/jxl/dec_ans.h

```diff
@@ -363,6 +363,23 @@ class ANSSymbolReader {
     return ret;
   }
 
+  // same but not inlined
+  template <bool uses_lz77>
+  size_t ReadHybridUintClustered(size_t ctx, BitReader* JXL_RESTRICT br) {
```

Comment:
Would be great to add JXL_NOINLINE. Otherwise templates are quite fluent on the solution whether to inline.

Reply:
Right, there's no guarantee that it will not be inlined in this case. It doesn't though, at least not with my version of clang and on my platform.